tests/sentry/api/endpoints/test_organization_codeowners_associations.py

```diff
@@ -105,19 +105,14 @@ def test_response_data_is_correct(self):
             in response.data[self.project_2.slug]["errors"]["teams_without_access"]
         )
 
-    def test_no_access(self):
+    def test_member_can_access(self):
         """
-        Tests that users without the 'org:integrations' scope (i.e. Members) cannot access this endpoint.
+        Tests that users without the 'org:read' scope (i.e. Members) can access this endpoint.
         """
         member = self.create_user("hernando@life.com")
         self.create_member(user=member, organization=self.organization, role="member")
```

Comment:
The creation of users and members in here are slightly different than the other `setUp` methods I have looked at:
https://github.com/getsentry/sentry/blob/c605f45a3ca0e585cd7ca1c5a0c286c0d61913e3/tests/sentry/api/endpoints/test_organization_code_mappings.py#L15
https://github.com/getsentry/sentry/blob/c605f45a3ca0e585cd7ca1c5a0c286c0d61913e3/tests/sentry/api/endpoints/test_organization_code_mappings.py#L23-L28

This is just an observation. It probably does not matter.